mod/gc_onboard/views/default/profile-steps/stepFive.php

```diff
@@ -13,9 +13,9 @@
 <?php echo elgg_view('avatar/upload', array('entity' =>  elgg_get_logged_in_user_entity())); ?>
 
 <div class="mrgn-bttm-md mrgn-tp-md pull-right">
-    <a href="<?php echo $profile_link; ?>" class="btn btn-primary btn-lg mrgn-tp-md mrgn-lft-sm" style="" title="" onclick="">
+    <button type="button" id="onboard-finish" class="btn btn-primary btn-lg mrgn-tp-md mrgn-lft-sm">
```

Comment:
Since this is actually a link, why not keep it that way and just clear out the onclick and still stuff?